test/Conversion/AIEVecToLLVM/test-pack.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: aie-opt %s --lower-aievec | FileCheck %s
+module {
+  func.func @test() {
+    %0 = llvm.mlir.undef: vector<16xi16>
+    %1 = aievec.pack %0 : vector<16xi16>, vector<16xi8>
+    return
+  }
+}
+// CHECK: %1 = llvm.call @llvm.aie.pack.v16int16(%0) : (vector<16xi16>) -> vector<16xi8>
```

Comment:
Here you are even using `%0` which could be anything, even something wrong. Either it matters, in which case you should capture it with a regex, or it doesn't, in which case you should ignore it with a regex. Like so:
```
// CHECK:    %[[UDEFV:.*]] = llvm.mlir.undef : vector<16xi16>
// CHECK:    %{{.*}} = llvm.call @llvm.aie.pack.v16in16(%[[UDEFV]]) : (vector<16xi16>) -> vector<16xi8>
```

or so:

```
// CHECK:    %{{.*}} = llvm.call @llvm.aie.pack.v16int16(%{{.*}})) : (vector<16xi16>) -> vector<16xi8>
```